conformance_pack/backup.sp

```diff
@@ -186,3 +206,63 @@ query "backup_recovery_point_min_retention_35_days" {
       aws_backup_recovery_point;
   EOQ
 }
+
+query "backup_plan_region_configured" {
+  sql = <<-EOQ
+    with count_plans as (
+      select
+        region,
+        account_id,
+        count(*) as count
+      from
+        aws_backup_plan
+      group by
+        region,
+        account_id
+    )  
+    select
+      'arn:' || r.partition || '::' || r.region || ':' || r.account_id as resource,
+      case
+        when cp.count > 0 then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when cp.count > 0 then cp.count || ' backup plan(s) exist in region ' || r.region || '.'
+        else 'No backup plans exist in region ' || r.region || '.'
+      end as reason
+      ${replace(local.common_dimensions_qualifier_sql, "__QUALIFIER__", "r.")}
+    from
+      aws_region as r
+      left join count_plans as cp on r.account_id = cp.account_id and r.region = cp.region;
+  EOQ
+}
+
+query "backup_vault_region_configured" {
+  sql = <<-EOQ
+    with count_vaults as (
+      select
+        region,
+        account_id,
+        count(*) as count
+      from
+        aws_backup_vault
+      group by
+        region,
+        account_id
+    )  
+    select
+      'arn:' || r.partition || '::' || r.region || ':' || r.account_id as resource,
+      case
+        when v.count > 0 then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when v.count > 0 then v.count || ' backup vault(s) exist in region ' || r.region || '.'
+        else 'No backup vault exists in region ' || r.region || '.'
```

Comment:
Do we need to add the region information? Won't the data be automatically available via the region common dimension?